container-setup/utils/bin/pause-hive-sync

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# OCM_CONTAINER_DOC: Adds the hive syncset-pause annotation to a cluster deployment
+
+# Annotation to pause sync
+ANNOTATION="hive.openshift.io/syncset-pause=\"true\""
```

Comment:
does this make it easier to read or do you prefer the escaping?

I guess you did this to be consistent to other parts of the script, so I won't feel bad on ignoring this nit
```suggestion
ANNOTATION='hive.openshift.io/syncset-pause="true"'
```